scripts/release/release-dev.js

```diff
@@ -21,19 +22,18 @@ const releaseDev = () => {
     const distPath = path.resolve(__dirname, '../../', 'packages', packageName, 'public', 'dist')
     runCommand('rimraf', [`${distPath}/**/*.map`])
     // This command remove the old version file in bucket
-    runCommand('aws', ['s3', 'rm', '--recursive', `s3://${bucketName}`])
-    // Sync new version to the bucket
-    runCommand('aws', [
-      's3',
-      'cp',
-      distPath,
-      `s3://${bucketName}`,
-      '--grants',
-      'read=uri=http://acs.amazonaws.com/groups/global/AllUsers',
-      '--recursive',
-      '--cache-control',
-      'max-age=31536000',
-    ])
+    const deleteResult = execSync(`aws s3 rm --recursive s3://${bucketName}`).toString()
+    console.log(deleteResult)
+    // cp all assert with cache-control 365 days exclude sw.js and index.html
+    const copyWithCache = execSync(
+      `aws s3 cp ${distPath} s3://${bucketName} --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers --recursive --exclude "index.html" --exclude "sw.js" --exclude "config.json" --cache-control "max-age=31536000"`,
+    ).toString()
+    console.log(copyWithCache)
+    // cp index.html and sw.js with no-cache control
+    const copyWithNoCache = execSync(
+      `aws s3 cp ${distPath} s3://${bucketName} --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers --recursive --exclude "*" --include "sw.js" --include "index.html" --include "config.json"`,
+    ).toString()
+    console.log(copyWithNoCache)
```

Comment:
Remove console.log

Reply:
This one will need to console to view the result after run